airflow/providers/amazon/aws/sensors/batch.py

```diff
@@ -87,17 +87,9 @@ def poke(self, context: Context) -> bool:
             return False
 
         if state == BatchClientHook.FAILURE_STATE:
-            # TODO: remove this if block when min_airflow_version is set to higher than 2.7.1
-            message = f"Batch sensor failed. AWS Batch job status: {state}"
-            if self.soft_fail:
-                raise AirflowSkipException(message)
-            raise AirflowException(message)
-
-        # TODO: remove this if block when min_airflow_version is set to higher than 2.7.1
-        message = f"Batch sensor failed. Unknown AWS Batch job status: {state}"
-        if self.soft_fail:
-            raise AirflowSkipException(message)
-        raise AirflowException(message)
+            raise AirflowException(f"Batch sensor failed. AWS Batch job status: {state}")
+
+        raise AirflowException(f"Batch sensor failed. Unknown AWS Batch job status: {state}")
```

Comment:
Just to prove I actually reviewed it :P  Feel free to reject.


```suggestion
     raise AirflowException(f"Batch sensor failed. {"Unknown " if state != BatchClientHook.FAILURE_STATE else ""} AWS Batch job status: {state}")